models/models.go

```diff
@@ -634,14 +637,14 @@ func (m *RouteManager) updateProvisioning(r *Route) error {
 }
 
 func (m *RouteManager) Destroy(guid string) error {
+	m.logger.Info("destroy-route", lager.Data{
```

Comment:
```
lsession := m.logger.Session("route-manager", lager.Data{"guid", guid})
lsession.Info("destroy-route")

// rest of the logging stuff should use `lsession` instead of `m.logger`
```